vignette/Validation.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "Comparison"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Comparison}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>", 
+  fig.path = "man/figures/README-",
+  fig.width = 9, 
+  fig.height = 9
+)
+```
+
+```{r setup}
+library(hypatia)
+library(ggplot2)
+library(tidyverse)
```

Comment:
Your vignette must not use any packages that are not listed in the Suggests (or Imports) of the DESCRIPTION, and tidyverse is not a package that should be depended on by other packages. What do you need from it? Please replace with imports to the needed packages (looks like tidyr, tibble and dplyr)

Given that you namespace qualify the hypatia calls, you don't need `library(hypatia)` either.